pkgs/games/rare/default.nix

```
@@ -13,13 +14,13 @@
 
 buildPythonApplication rec {
   pname = "rare";
-  version = "1.10.3";
+  version = "1.10.6";
 
   src = fetchFromGitHub {
     owner = "Dummerle";
```

```suggestion
    owner = "RareDevs";
```
Based on the change of the `meta.homepage`